### ISSUE TITLE
Investigate adding HTML linting for front-end accessibility

### ISSUE THREAD

**Issue by governmentSponsored** [US-Trustee-Program/Bankruptcy-Oversight-Support-Systems#512]
Want to have our IDEs indicate if/when HTML/FE react code we have written is in violation of accessibility standards.

**Comment by btposey** [comment]
The problem with this is we write JSX which is compiled into HTML by React. We do not write HTML directly.

**Comment by jharcey-flexion** [comment]
Deleted from Jira
